### DISCUSSION TITLE
New "About Organized" webpage is released!

### DISCUSSION THREAD

**Discussion by ux-git** [sws2apps/organized-app#1277] [Announcements]
# New "About Organized" page is live! 🚀

We're happy to announce that our new **[About Organized](https://about.organized-app.com)** page is now live! This dedicated web page provides an overview of the Organized app, including its goals, features, and benefits for users. It's the perfect place to learn more about how Organized can improve your congregation's scheduling, reporting and simplify collaboration.

## Why is this page useful:

- **Learn more:** Find out everything you need to know about Organized, and maybe discover something new along the way.
- **Share with others:** Easily share the page link with your elders, friends, or congregation members to introduce them to Organized in an accessible way.
- **FAQs:** Get answers to all your questions, such as how it works, how secure it is, and how to start using it.

![2024-03-1821-17-15-ezgif com-video-to-gif-converter](https://github.com/sws2apps/organized-app/assets/80993061/80e1da27-7bc9-476b-8f2c-25c8d16d043f)
_Preview of the website_

## Want to translate? 🌍

Want to read the page in your language and share with local brotherhood? You can help us translate it over on our [Crowdin project](https://crowdin.com/project/organized). The `About Organized website` folder contains all the strings for this website, and your contributions are highly appreciated. 

> Some strings are limited in characters due to space constraints. In these cases, feel free to rephrase sentences and words while preserving the original idea.

### Thank you for your continued support and enthusiasm for the app! Let's make Organized better and more accessible together!

Best regards,  
Your sws2apps team

**Comment by fschein** [comment]
Hi! I revised everything on Crowdin, but the page isn’t switching to Portuguese, neither PT nor BR.

**Reply by mrtavor** [reply]
Hi @fschein , I have already started working on a solution. Thanks for letting us know 🙂

**Reply by mrtavor** [reply]
@fschein Now it works great. If you still have any issues, please let us know, we’d be happy to help

**Reply by fschein** [reply]
That was quick. Thanks!
